homeassistant/components/syncthru/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "This device has already been configured"
+    },
+    "error": {
+      "invalid_url": "Invalid URL",
+      "syncthru_not_supported": "Device does not support SyncThru",
+      "unknown_state": "Printer state unknown, verify URL and network connectivity"
+    },
+    "flow_title": "Samsung SyncThru Printer: {name}",
+    "step": {
+      "confirm": {
+        "data": {
+          "name": "Name",
+          "url": "Web interface URL"
+        }
+      },
+      "user": {
+        "data": {
+          "name": "Name",
+          "url": "Web interface URL"
```

Comment:
Why are these duplicate with `confirm` ?

Reply:
I tried to reuse all of the `user` step code when clicking configure on a discovered printer, however for some reason the form always receives empty values if I do that, so I can't get the form's default values set to the ones from discovery. Using a separate `confirm` step does not have that problem.

I also did not know how to refer to strings of a step from another, but TIL, changed now.